mesheryctl/internal/cli/root/app/offboard.go

```diff
@@ -53,27 +53,28 @@ mesheryctl app offboard -f [filepath]
 			return errors.Wrap(err, "error processing config")
 		}
 
+		var urlString = mctlCfg.GetBaseMesheryURL()
 		app := ""
 		isID := false
 		if len(args) > 0 {
-			app, isID, err = utils.ValidId(args[0], "application")
+			app, isID, err = utils.ValidId(urlString, args[0], "application")
```

Comment:
Unless we are pervasively referring to `mctlCfg.GetBaseMesheryURL()` as `urlString` in all commands, I think this change hurts readability. 

Unless there is some other reason to use a variable to reference the value of `mctlCfg.GetBaseMesheryURL()`, I would like to see us refer to the function directly. `urlString` strikes me as a reference to an App/Filter location, not as the Meshery Server API location.

Reply:
I thought that calling it again and again as it is needed at multiple places would lead into increase in execution time as it is reading the config file so i thought it would be good to call it only once

Reply:
@suhail34 Can you please make the change back to calling `mctlCfg.GetBaseMesheryURL()` in the functions. Instead of in the command.